packages/components/src/components/ScalarMarkdown/ScalarMarkdown.vue

```diff
@@ -31,9 +32,18 @@ onServerPrefetch(async () => await sleep(1))
 <template>
   <div
     class="markdown"
+    :class="{ collapsed: !isCollapsed }"
     v-html="html" />
 </template>
 
 <style>
 @import '@scalar/code-highlight/css/markdown.css';
+
```

Comment:
Can we have two style tags here? One for the Markdown CSS and one `scoped` for the `.collapsed` CSS?

Reply:
oh great catch i didnt see that!